docs/fidelity-bonds.md

```diff
@@ -113,8 +152,109 @@ coins using `python3 wallet-tool.py <walletname> -m 0 freeze`.
 Once unfrozen and untimelocked the coins can be spent normally with the scripts
 `sendpayment.py`, `tumber.py`, or yield generator.
```

Comment:
I think it makes sense to add a note here: "NB You cannot export the private keys (which is *always* disadvised, anyway) of timelocked addresses to any other wallets, as they use custom scripts. You must spend them from Joinmarket itself." (since we don't have any fancy wallet descriptor etc. stuff yet).